compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/input/internal/LegacyPlatformTextInputServiceAdapter.kt

```diff
@@ -26,11 +26,12 @@ import androidx.compose.ui.platform.PlatformTextInputSession
 import androidx.compose.ui.platform.SoftwareKeyboardController
 import androidx.compose.ui.platform.ViewConfiguration
 import androidx.compose.ui.text.input.PlatformTextInputService
+import androidx.compose.ui.text.input.TextInputService
 import kotlinx.coroutines.Job
 
 @Composable
-internal expect fun legacyPlatformTextInputServiceAdapter():
-    LegacyPlatformTextInputServiceAdapter
```

Comment:
Why should it be combined into a single function/pair? Isn't it independent entities?

Reply:
on our platforms - LegacyPlatformTextInputServiceAdapter depends TextInputService
on Android - TextInputService depends LegacyPlatformTextInputServiceAdapter.

It can only be implemented via pair